From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.com>
Date: Tue, 16 Aug 2022 14:10:39 +0200
Subject: [PATCH] github workflows: build-and-unittest.yaml: add libedit-dev

This is is required after switching from libreadline to libedit.

Signed-off-by: Martin Wilck <mwilck@suse.com>
Signed-off-by: Benjamin Marzinski <bmarzins@redhat.com>
---
 .github/workflows/build-and-unittest.yaml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/.github/workflows/build-and-unittest.yaml b/.github/workflows/build-and-unittest.yaml
index 7ff45842..1ab0d36c 100644
--- a/.github/workflows/build-and-unittest.yaml
+++ b/.github/workflows/build-and-unittest.yaml
@@ -27,7 +27,7 @@ jobs:
           sudo apt-get install --yes gcc
           make perl-base pkg-config valgrind
           libdevmapper-dev libreadline-dev libaio-dev libsystemd-dev
-          libudev-dev libjson-c-dev liburcu-dev libcmocka-dev
+          libudev-dev libjson-c-dev liburcu-dev libcmocka-dev libedit-dev
       - name: build
         run: make -O -j$(grep -c ^processor /proc/cpuinfo)
       - name: test
@@ -55,7 +55,7 @@ jobs:
           sudo apt-get install --yes gcc-10
           make perl-base pkg-config valgrind
           libdevmapper-dev libreadline-dev libaio-dev libsystemd-dev
-          libudev-dev libjson-c-dev liburcu-dev libcmocka-dev
+          libudev-dev libjson-c-dev liburcu-dev libcmocka-dev libedit-dev
       - name: set CC
         run: echo CC=gcc-10 >> $GITHUB_ENV
       - name: build
@@ -85,7 +85,7 @@ jobs:
           sudo apt-get install --yes clang
           make perl-base pkg-config valgrind
           libdevmapper-dev libreadline-dev libaio-dev libsystemd-dev
-          libudev-dev libjson-c-dev liburcu-dev libcmocka-dev
+          libudev-dev libjson-c-dev liburcu-dev libcmocka-dev libedit-dev
       - name: set CC
         run: echo CC=clang >> $GITHUB_ENV
       - name: build
